### PR TITLE
New CTA banner for reco empty state

### PR DESCRIPTION
**Pull request by stkem**
- no more “beta” and redo count

@yipingliao

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/recos/recos.styl [hunk 1]
```diff
@@ -10,15 +10,6 @@
 
   .kf-main-title
     position: relative
-    &::after
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/recos/recos.styl:13]: Yay!

File: kifi-backend/modules/shoebox/angular/src/recos/recos.styl [hunk 2]
```diff
@@ -126,6 +117,39 @@
...
+  line-height: 150%
+  width: 40%
+  padding: 25px 0px 25px 25px
+
+.kf-reco-cta-image
+  display: inline-block
+  width: 28%
+  padding: 5% 10%
+  float: right
+
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/recos/recos.styl:139]: Remove one newline.

File: kifi-backend/modules/shoebox/angular/src/recos/recos.styl [hunk 3]
```diff
@@ -126,6 +117,39 @@
...
+
+.kf-reco-cta:after
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/recos/recos.styl]: Use `::after`

File: kifi-backend/modules/shoebox/angular/src/recos/recosView.tpl.html [hunk 4]
```diff
@@ -5,18 +5,14 @@
...
   </header>
 
-  <div class="kf-reco kf-reco-info" ng-if="recosState === 'noRecos'" ng-hide="initialCardClosed">
-    <span class="kf-reco-icon kf-reco-close-x" ng-click="closeInitialCard()"></span>
-    <h4>Turn on Kifi Recommendations</h4>
-    <p>Kifi works by recommending things based on what you've kept. We'll cater to your interests over time but we need at least a dozen good articles kept to get started. It's quick to get started if you <a href="javascript:" ng-click="importBookmarks()">import bookmarks</a>, and even better if you <a href="/invite">connect with friends</a>.
-    <p>In the meantime, enjoy popular things being kept on Kifi. If you keep some of the ones you like it will help us give you personalized recommendations faster.</p>
-    <a href="http://support.kifi.com">Learn more</a>
+  <div class="kf-reco kf-reco-info kf-reco-cta" ng-if="recosState === 'noRecos'" ng-hide="initialCardClosed">
+    <span class="kf-reco-cta-text"> To get started, simply click the follow button on some of the libraries below </span>
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/recos/recosView.tpl.html]: Spaces around the cta text ("To get started...") intentional?

File: kifi-backend/modules/shoebox/angular/src/recos/recosView.tpl.html [hunk 5]
```diff
@@ -5,18 +5,14 @@
       <h1 class="kf-main-title">Recommendations</h1>
       <div class="kf-subtitle">
         <div class="kf-subtitle-text" ng-show="loading">Loading...</div>
-        <div class="kf-subtitle-text" ng-hide="loading">Showing {{recos.length}} recommendations</div>
       </div>
     </div>
     <div kf-reco-recency-slider ng-if="recosState === 'hasRecos'" get-more="getMore(recency)"></div>
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/recos/recosView.tpl.html:10]: Just curious: do we have any data on whether folks are using this yet? I don't even remember if we put in tracking for this...

**Comment by yipingliao** [comment]
:+1:

File: kifi-backend/modules/shoebox/angular/src/recos/recosView.tpl.html (hunk 5 above)
Comment (stkem) [inline kifi-backend/modules/shoebox/angular/src/recos/recosView.tpl.html:10]: If you are talking about the slider then no, we don't have tracking. It's only a matter of time before that goes away, imho.

File: kifi-backend/modules/shoebox/angular/src/recos/recosView.tpl.html (hunk 4 above)
Comment (stkem) [inline kifi-backend/modules/shoebox/angular/src/recos/recosView.tpl.html]: Nope. Fixing.

**Pull request merged**